templates/dart/lib/src/client_mixin.dart.twig

```diff
@@ -40,15 +40,15 @@ class ClientMixin {
     } else if (method == HttpMethod.get) {
       final encoded = <String, dynamic>{};
       if (params.isNotEmpty) {
-        params.keys.forEach((key) {
+        for(var key in params.keys) {
```

Comment:
Why a for loop when you could use a map? (Map is faster and resource efficient).

Reply:
updated